### PR TITLE
docs: migrate handbook to docs site

### PR DESCRIPTION
**Pull request by tataihono**
# Description

Please include a summary of the change and which todo is completed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- depends on https://github.com/JesusFilm/core/pull/1041
- [Migrate handbook to docs site](https://3.basecamp.com/3105655/buckets/29689605/todos/5521093770)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] [Handbook matches actual practices](https://core-docs-git-22-08-tn-docs-migrate-handbook-jesusfilm.vercel.app/docs/team/)

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged into main

### CODE REVIEW

File: apps/docs/docs/05-team/05-how-we-work.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+First, there’s the daily question of What did you work on today?, which supplies the nitty gritty details, but as a personal narrative. They’re a great conversation starter if you see someone working on something you either care about or want to learn more about. Please do use them as such! You’re obliged to answer this question at least twice a week when you’re not out.
+
+Second, there’s the weekly question of What will you be working on this week?, which details your intentions for the coming week. Everyone is obliged to answer this question when they’re not out.
+
+Third, there are the heartbeats. These are the team versions of What did you work on this cycle? This is where we summarize and celebrate the work that’s been done. Every team lead is obliged to write, or designate someone on the team to write, this account one week after a cycle has ended.
+
+Fourth, and finally, there are the kickoffs. These are the team version of What are you going to work on next cycle? This is where the plan for the coming four weeks is presented. Every team lead is obliged to write, or designate someone on the team to write, this account before the start of the new cycle.
```
Comment (csiyang) [inline apps/docs/docs/05-team/05-how-we-work.md:25]:
```suggestion
Fourth, and finally, there are the kickoffs. These are where the plan for the coming four weeks is presented. Every team lead is obliged to write, or designate someone on the team to write, this account before the start of the new cycle.
```

File: apps/docs/docs/05-team/06-making-a-career/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Making a Career
+
+Advancing your career at Jesus Film doesn’t mean giving up on your craft. Whether you work in programming, design, ops, support, or whatever, you can become better at the work itself and level-up that way. This is especially important since we’re a relatively small team with just two layers of managerial cake: executives and team managers. And both the [executives and managers](../our-responsibilities) still spend the majority of their time doing actual product work!
+
+Within each of our job functions, we’ve mapped our trajectory of mastery to five different levels. That title structure is shared amongst all departments, but the particulars of what characterizes one level from another will of course be different. Here’s an example of the titles for product engineering:
+
+- Graduate Product Engineer
```
Comment (csiyang) [inline apps/docs/docs/05-team/06-making-a-career/index.md:7]:
```suggestion
- Junior Product Engineer
```

File: apps/docs/docs/05-team/05-how-we-work.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## Cooldown
+
+In between each cycle, we spend one week cooling down. That’s the time to deal with bugs or smaller issues that come up, write up what we worked on, and figure out what we should tackle next. It’s sometimes tempting to simply extend the cycles into the cooldown period to fit in more work. But the goal is to resist this temptation. Yes, sometimes a little spill-over will happen, but it’s helpful to think about the end of the normal cycle as "pencils down". That means that by week 4 of a normal cycle, we should be winding down, getting ready to launch, make sure QA is lined up, and all the other work that happens during and after the launch of new projects.
```
[the remark this answers is not in the document]
Comment (GeronimoJohn) [reply apps/docs/docs/05-team/05-how-we-work.md:13]: Though it's also 1 week cooling down on the actual docs
https://core-docs-git-22-08-tn-docs-migrate-handbook-jesusfilm.vercel.app/docs/team/how-we-work

File: apps/docs/docs/05-team/07-our-habits.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Our Habits
+
+## Meetups
+
+Twice a year, our team gathers at a ministry hub for a week. Your first meet-up will be overwhelming; there are a lot of people to meet, most or all of whom you’ve only chatted with virtually. People tend to arrive on the Sunday before the meet-up, but ’campers who travel longer distances arrive earlier to beat the jet lag. Jesus Film books accomodation nearby, so you don’t need to worry about finding a place to stay.
```
Comment (csiyang) [inline apps/docs/docs/05-team/07-our-habits.md:5]: This is referring to when all JF SOL team gathers?

```suggestion
Once a year, our team gathers at a ministry hub for a week. Your first meet-up will be overwhelming; there are a lot of people to meet, most or all of whom you’ve only chatted with virtually. People tend to arrive on the Sunday before the meet-up, but ’campers who travel longer distances arrive earlier to beat the jet lag. Jesus Film books accomodation nearby, so you don’t need to worry about finding a place to stay.
```

File: apps/docs/docs/05-team/08-our-responsibilities.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+Jesus Film has three tiers of responsibility within our job functions: Executive, Manager, and Individual. Here’s how we break down each level’s duties.
+
+## Executive
+
+- organization and product strategy
+  - organization: structure, finances, marketing/publishing, evaluating/adjusting core values and ethics
+  - product: values, long-term direction, short-term (cycle) direction, marketing
+- product work and communication
+  - making personal contributions to product work (programming and design)
+  - writing heartbeats and kick-offs (with input from team); providing status updates in between as needed
```
Comment (csiyang) [inline apps/docs/docs/05-team/08-our-responsibilities.md:12]:
```suggestion
  - writing standup updates and kick-offs (with input from team); providing status updates in between as needed
```

File: apps/docs/docs/05-team/08-our-responsibilities.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  - spot-checking Managers and stepping in as needed; requesting status updates from teams
+
+## Manager
+
+- team strategy
+  - interpreting cycle focus from Executives
+  - driver/delegator of work; "team lead"; ultimately responsible for team’s output
+- product work and communication
+  - making personal contributions to product work (programming, design, Ops, customer happiness)
+  - writing heartbeats and kick-offs (with input from team) or contributing to Executive write-ups; providing status updates in between as needed
```
Comment (csiyang) [inline apps/docs/docs/05-team/08-our-responsibilities.md:28]:
```suggestion
  - writing standup updates and kick-offs (with input from team) or contributing to Executive write-ups; providing status updates in between as needed
```

File: apps/docs/docs/05-team/08-our-responsibilities.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  - maintaining general cohesion of team (via 1:1s, group hangs, etc.)
+  - anticipating and addressing morale issues
+  - keeping the pulse of the emotional well-being of team including workload, attachment to work, on the lookout for fatigue or burnout, team advocate
+  - translating team feedback on overall organization direction and morale to Executives
+
+## Individual
+
+- product work and communication
+  - making personal contributions to product work (programming, design, Ops, customer happiness)
+  - contributing to heartbeats and kick-offs; providing status updates in between as needed
```
Comment (csiyang) [inline apps/docs/docs/05-team/08-our-responsibilities.md:44]:
```suggestion
  - contributing to standup updates and kick-offs; providing status updates in between as needed
```

File: apps/docs/docs/05-team/09-international-travel-guide.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+  - [Android](https://support.google.com/chrome/answer/95647?co=GENIE.Platform%3DAndroid&hl=en): Chrome > Settings > Privacy > Clear browsing data. Check Cookies and site data; uncheck all the other items. > Clear data.
+- Jesus Film source code.
+  - Make an encrypted archive on an external hard drive of your Jesus Film git repos so you can download and restore it after you’re through customs. Here are [instructions](https://support.apple.com/en-us/HT201250) for doing that if you're running a Mac OS. Please perform an equally secure file encryption for Windows or Linux OS.
+  - Delete all Jesus Film source code from your laptop.
+- Other sensitive documents.
+  - Same deal as source code. Make an encrypted backup on your external drive and remove from your laptop. Please refrain from using iCloud for any files/repos.
+- Touch ID (optional)
+  - Unlocking your device can be compelled at US border crossings, but other countries may be less intrusive, so it may make sense to make your device a little harder to unlock.
+  - Disable Touch ID on your phone, tablet, and laptop. Require a passcode/password to unlock.
+
```
Comment (csiyang) [inline apps/docs/docs/05-team/09-international-travel-guide.md:73]: Add instructions for WARP?

**Review by GeronimoJohn — APPROVED** [review]

**Review by csiyang — CHANGES_REQUESTED** [review]

File: apps/docs/docs/05-team/08-our-responsibilities.md (hunk 5 above)
Comment (tataihono) [reply apps/docs/docs/05-team/08-our-responsibilities.md:12]: No. heartbeats are a thing they need to write.

File: apps/docs/docs/05-team/08-our-responsibilities.md (hunk 6 above)
Comment (tataihono) [reply apps/docs/docs/05-team/08-our-responsibilities.md:28]: No. heartbeats are a thing they need to write.

File: apps/docs/docs/05-team/08-our-responsibilities.md (hunk 7 above)
Comment (tataihono) [reply apps/docs/docs/05-team/08-our-responsibilities.md:44]: No. heartbeats are a thing they need to write.

File: apps/docs/docs/05-team/09-international-travel-guide.md (hunk 8 above)
Comment (tataihono) [reply apps/docs/docs/05-team/09-international-travel-guide.md:73]: Still deciding if thats what we want to use in the long term. We may end up using AWS for this.

File: apps/docs/docs/05-team/07-our-habits.md (hunk 4 above)
Comment (tataihono) [reply apps/docs/docs/05-team/07-our-habits.md:5]: Done.

**Review by csiyang — APPROVED** [review]

File: apps/docs/docs/05-team/06-making-a-career/02-titles-for-designers.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+## Lead Designer
+
+- Work happens completely autonomously without regular need for review.
+- Can run point on large batch projects, taking them from concept to shipping as the sole designer (alongside a engineer).
+- Can lead a small department or longer-term initiative.
+- Helps set and maintain professional standards for the entire organization.
+- Makes regular, material contributions to the look-and-feel of our general aesthetic and brand.
+- Usually at least 8-12 years of experience being a professional designer.
+- Typically at least 3-5 years experience at Jesus Film internalizing how we work.
+- WEB: Fully capable of writing modern HTML/CSS/JS + the ability to make moderate progress in Rails.
```
Comment (storyworks) [inline apps/docs/docs/05-team/06-making-a-career/02-titles-for-designers.md:42]: Rails!?

Comment (tataihono) [reply apps/docs/docs/05-team/06-making-a-career/02-titles-for-designers.md:42]: fixed.

**Review by storyworks — APPROVED** [review]

**Pull request merged**